analytics/views.py

```diff
@@ -1193,6 +1193,9 @@ def support(request: HttpRequest) -> HttpResponse:
                 downgrade_now_without_creating_additional_invoices(realm)
                 voided_invoices_count = void_all_open_invoices(realm)
                 context["message"] = f"{realm.string_id} downgraded and voided {voided_invoices_count} open invoices"
+            event_time = timezone_now()
+            RealmAuditLog.objects.create(realm=realm, acting_user=request.user, event_type=RealmAuditLog.REALM_DOWNGRADE_METHOD_CHANGED,
+                                         event_time=event_time)
```

Comment:
Don't add any logging here

Add logging in `downgrade_now_without_creating_additional_invoices`, `downgrade_at_the_end_of_billing_cycle, `downgrade_at_the_end_of_billing_cycle` etc by making them take `acting_user` as an additional argument.